pkg/transformer/openshift/openshift.go

```diff
@@ -200,6 +200,13 @@ func (o *OpenShift) initImageStream(name string, service kobject.ServiceConfig,
 // initBuildConfig initialize Openshifts BuildConfig Object
 func initBuildConfig(name string, service kobject.ServiceConfig, repo string, branch string) (*buildapi.BuildConfig, error) {
 	contextDir, err := getAbsBuildContext(service.Build)
+	envList := []kapi.EnvVar{}
```

Comment:
as https://github.com/kubernetes-incubator/kompose/pull/602 puts `kapi` instead of `api`